queries/usersQueries.js

```diff
@@ -1,18 +1,24 @@
-const usersData = require('../usersData.json');
+const db = require("../db/dbConfig");
 
-const idToUserIndex = usersData.reduce((index, user) => {
-  const { id } = user;
-  index[id] = user;
-  return index;
-}, {});
+const getAllUsers = async () => {
+  try {
+    const users = await db.any("SELECT * FROM users");
+    return users;
+  } catch (err) {
+    return err;
```

Comment:
Nice! It is so important to return something from the catch block. One thing to consider is how to return it in a way so that the client will know that there was an error. Because you are catching the error here, your controller code will never go into the `catch` block and that means the controller code needs to check what it gets from the query for errors.